.eslintrc.js

```diff
@@ -250,6 +250,13 @@ module.exports = {
 				],
 			},
 		],
+		'@typescript-eslint/consistent-type-imports': [
+			'error',
+			{
+				prefer: 'type-imports',
+				disallowTypeAnnotations: false,
```

Comment:
We frequently import type annotations via JSDoc in Gutenberg, so we need to keep this setting off :)